src/azure-cli/azure/cli/command_modules/signalr/tests/latest/test_signalr_commands.py

```diff
@@ -47,10 +61,12 @@ def test_signalr_commands(self, resource_group):
             self.check('provisioningState', 'Succeeded'),
             self.check('sku.name', '{sku}'),
             self.check('sku.capacity', '{unit_count}'),
+            self.check('features[0].value', '{service_mode}'),
+            self.check('cors.allowedOrigins', allowed_origins),
             self.exists('hostName'),
             self.exists('publicPort'),
             self.exists('serverPort'),
-            self.exists('externalIp')
+            self.exists('externalIp'),
```

Comment:
I think this coma is not necessary, but that' sminor

Reply:
Generally, I add coma in the last line so that git will only show one-line change if I add new lines in the future.